src/index.js

```diff
@@ -133,3 +126,171 @@ export class WealthscopeSdk {
     }
   }
 }
+
+export class WealthscopeApiClient {
+  /**
+   * Constructor for the Wealthscope Client API.
+   * Stores the authorization token for the current session, and the base URl.
```

Comment:
1) The CONSTRUCTOR itself doesn't store the authorization token. The OBJECT INSTANCE stores the token. the constructor only initializes it to `null`
2) "URL" should be all caps or all lowercase